### DISCUSSION TITLE
Not  programming...

### DISCUSSION THREAD

**Discussion by JVAN43** [community/community#146768] [Programming Help]
### Body

I'm not sure this falls into programming or implementation but here is where I am.

I am a novice, but trying to install an edge driver to allow my smartthings hub to interact with my tempest weather station. (https://github.com/toddaustin07/edgeWeather). I think I have followed the directions and the driver is on the smartthings hub, but no information is going back and forth.

I need some direction on where to go next. I have attached pix of where I am. Any help would be great. I have reached out to Todd, but no help so far.

Thanks

![20241209_094034](https://github.com/user-attachments/assets/bfefc0e4-1b8c-4953-9947-dd8f9cd4d8e2)
![Current Weather URL](https://github.com/user-attachments/assets/238e1e9f-860c-40e3-8c0c-9cc3e4edcaaa)
![Forecast URL](https://github.com/user-attachments/assets/69d631a6-a780-4c65-b04b-54d155392326)
![Screenshot_20241209_094112_SmartThings](https://github.com/user-attachments/assets/c75e608b-6da9-40b1-ad37-28f4c93d2e2b)

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by mecodeatlas** [comment]
Hi @JVAN43,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as the [SmartThings Support](https://support.smartthings.com/hc/en-us/requests/new). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
